backend/src/achievements/achievements.controller.ts

```diff
@@ -0,0 +1,56 @@
+import { AchievementsService } from './achievements.service';
+import { ApiResponse, ApiTags } from '@nestjs/swagger';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  NotFoundException,
+  Param,
+  Post,
+} from '@nestjs/common';
+import { Public } from 'src/auth/auth.public.decorator';
+import { AchievementDto } from './dto/achievement.dto';
+import { ResponseAchievementDto } from './dto/response-achievement.dto';
+import { DeleteResult } from 'typeorm';
+
+@ApiTags('achievements')
```

Comment:
```suggestion
@ApiBearerAuth()
@ApiTags('achievements')
```
This will tell Swagger that the Bearer Token must be sent with requests to routes not marked `@Public`. It will be necessary if you decide to make the create/delete routes protected (see below)